universal-application-tool-0.0.1/app/services/program/ProgramService.java

```diff
@@ -86,6 +86,18 @@
   ErrorAnd<ProgramDefinition, CiviFormError> addBlockToProgram(long programId)
       throws ProgramNotFoundException;
 
+  /**
+   * Adds an empty repeated {@link BlockDefinition} to the given program.
+   *
+   * @param programId the ID of the program to update
+   * @param repeaterId an reference to a repeater block
+   * @return the {@link ProgramDefinition} that was updated if succeeded, or a set of errors with
+   *     the unmodified program definition if failed
+   * @throws ProgramNotFoundException when programId does not correspond to a real Program.
+   */
+  ErrorAnd<ProgramDefinition, CiviFormError> addRepeatedBlockToProgram(
+      long programId, long repeaterId) throws ProgramNotFoundException;
```

Comment:
Nit(blocking): could we call this repeaterBlockId to distinguish it from repeater(Question)Id?

Reply:
done